### PR TITLE
Exclude coverage outside `lib/`

### PR DESCRIPTION
**Pull request by ehuelsmann**
# Description

Change in the build/CI system

## Type of change

No code changes.

### CODE REVIEW

**Review by jonasbn — APPROVED** [review]
Lets try it 👍🏻

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ jobs:
         run: |
           cd build
           [ -d xt ] && prove --timer --lib --recurse --jobs $(nproc) --shuffle xt
-          HARNESS_PERL_SWITCHES=-MDevel::Cover=+ignore,^t/ prove --timer --lib --recurse --jobs $(nproc) --shuffle t/
+          HARNESS_PERL_SWITCHES=-MDevel::Cover=+ignore,^t/,+ignore,^/ prove --timer --lib --recurse --jobs $(nproc) --shuffle t/
```
Comment (jonasbn) [inline .github/workflows/ci.yml:50]: Perhaps this should be expressed as:

```suggest
HARNESS_PERL_SWITCHES=-MDevel::Cover=+ignore,^t/,^/ prove --timer --lib --recurse --jobs $(nproc) --shuffle t/
```

**Comment by ehuelsmann** [comment]
I think this consistently hits files in `lib/` instead of in `../lib` sometimes.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ehuelsmann) [reply .github/workflows/ci.yml]: Yes, that'd be better. However, I think the problem came from an entirely different angle than I originally expected. (Thanks to the people in #perl on OFTC for taking a peek at our code base!) So, I have a completely new proposal now.

**Comment by ehuelsmann** [comment]
The fact that the total coverage drops compared to prior results, has two causes:

* Persister/DBI/ExtraData is now included under lib/ at only 12% coverage (lowering the coverage of lib/Workflow/Persister and subdirectories)
* Under "/", there were results from t/TestUtil.pm and t/PersisterOtherDBIFields.pm, both of which had great coverage, but which are now both excluded from the coverage result due to the ^t/ ignore instruction

**Review by jonasbn — APPROVED** [review]
Looks good

**Pull request merged**